src/libraries/System.Linq.Expressions/src/System/Dynamic/DynamicObject.cs

```diff
@@ -113,7 +114,7 @@ public virtual bool TryConvert(ConvertBinder binder, out object result)
         /// <param name="result">The created instance.</param>
         /// <returns>true if the operation is complete, false if the call site should determine behavior.</returns>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1007:UseGenericsWhereAppropriate")]
-        public virtual bool TryCreateInstance(CreateInstanceBinder binder, object[] args, out object result)
+        public virtual bool TryCreateInstance(CreateInstanceBinder binder, object[]? args, out object? result)
```

Comment:
>out [](start = 91, length = 3)

`[NotNullWhen(true)]`?

Reply:
i didn't found an implementation for this one, anyways will add that